XCUITests/HomePageSettingsUITest.swift

```diff
@@ -122,6 +122,19 @@ class HomePageSettingsUITests: BaseTestCase {
 
         XCTAssertEqual("Enter a webpage", value as! String)
     }*/
+    
+    func testSetFirefoxHomeAsHome() {
+        navigator.goto(HomeSettings)
+        navigator.performAction(Action.SelectHomeAsHistoryPage)
+        app.navigationBars.buttons.element(boundBy: 0).tap()
```

Comment:
Same comment as in the other PR about the back actions to be handled by the graph